backend/services/operations.go

```diff
@@ -352,3 +417,44 @@ func SanitizeOperationSlug(slug string) string {
 		"-",
 	)
 }
+
+var getDataFromEvidence string = `
+	SELECT
+		slug,
+		operation_id,
+		count(evidence.id) AS count
+	FROM
+		evidence
+		LEFT JOIN users ON evidence.operator_id = users.id`
+
+var GetTopContributorsForEachOperation string = fmt.Sprintf(`
+	SELECT
+		t1.*
+	FROM (%s
+	GROUP BY
+		operation_id,
+		users.id) t1
+		LEFT JOIN (
+			%s	
+			GROUP BY
+				operation_id,
+				users.id) t2 ON t1.operation_id = t2.operation_id
+		AND t1.count < t2.count
+	WHERE
+		t2.count IS NULL`, getDataFromEvidence, getDataFromEvidence)
+
+var getCountsFromEvidence string = `
+SELECT operation_id,
+	COUNT(CASE WHEN content_type = "image" THEN 1 END) image_count,
+	COUNT(CASE WHEN content_type = "codeblock" THEN 1 END) codeblock_count,
+	COUNT(CASE WHEN content_type = "terminal-recording" THEN 1 END) recording_count,
+	COUNT(CASE WHEN content_type = "event" THEN 1 END) event_count,
+	COUNT(CASE WHEN content_type = "http-request-cycle" THEN 1 END) har_count
+FROM
+	evidence
```

Comment:
There are some limitations around what kind of operations you can do when you use mysql and aurora (e.g. I believe common table expressions may not be supported). I can't tell if this is allowed or not. I think you'll find out quickly when a test deployment occurs.

Reply:
oooh that's good to know